src/Cake.Issues/Serialization/SerializableIssueV3.cs

```diff
@@ -95,5 +96,9 @@ public int Version
         /// <inheritdoc cref="IIssue.Run" />
         [DataMember]
         public string Run { get; set; }
+
+        /// <inheritdoc cref="IIssue.AdditionalInformation" />
+        [DataMember]
+        public Dictionary<string, string> AdditionalInformation { get; set; }
```

Comment:
You can't change existing serialization format version 3, which was deployed with Cake.Issues 0.9.0. If changes are made to serialization format it should get a new version, 4 in this case. Serialization format version 4 is what will come with Cake.Issues vNext (0.10.0).

Reply:
@pascalberger Gotha... I changed it. Better? :)